test/e2e-go/features/transactions/accountv2_test.go

```diff
@@ -207,14 +209,17 @@ int 1
 	a.NoError(err)
 	round, err = client.CurrentRound()
 	a.NoError(err)
-	_, err = client.BroadcastTransaction(signedTxn)
+	txid, err = client.BroadcastTransaction(signedTxn)
 	a.NoError(err)
 	_, err = client.WaitForRound(round + 3)
 	a.NoError(err)
 	// Ensure the txn committed
 	resp, err := client.GetPendingTransactions(2)
 	a.NoError(err)
 	a.Equal(uint64(0), resp.TotalTxns)
```

Comment:
Checking the pending transactions is good, but it's not sufficiently deterministic. If the node were to reset, the transaction pool would get clear out, and the transaction might or might not be there. Checking the client.TransactionInformation is much more robust.

Reply:
Just remove GetPendingTransactions then

Reply:
It wouldn't be adding it, but since it's already there, let's keep it for now so that this test would also provide a coverage for that method ?